Gems/Prefab/PrefabBuilder/PrefabBuilderComponent.cpp

```diff
@@ -44,10 +44,12 @@ namespace AZ::Prefab
 
         AssetBuilderSDK::AssetBuilderDesc builderDesc;
         builderDesc.m_name = "Prefab Builder";
+        builderDesc.m_version = 1;
```

Comment:
if it's likely we want to bump the class serialize version and this builder description version at the same time, it might be a good idea to leave a comment to remind future devs to not forget to do that

Reply:
Good call, I've made it a const and added a comment.